mllib/src/test/scala/org/apache/spark/ml/clustering/GaussianMixtureSuite.scala

```diff
@@ -142,6 +142,32 @@ class GaussianMixtureSuite extends SparkFunSuite with MLlibTestSparkContext
     assert(!model.hasSummary)
   }
 
+  test("check LogLikelihood") {
```

Comment:
I think we don't need to bother a separate test, you can add check for ```logLikelihood``` on the existing test(multivariate data and check againt R mvnormalmixEM) which is equivalent to what you wrote but with more reasonable dataset.